index.html

```diff
@@ -3,7 +3,129 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script
+      src="https://kit.fontawesome.com/39da8831d4.js"
+      crossorigin="anonymous"
+    ></script>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+    </style>
+
     <title>Document</title>
   </head>
-  <body></body>
+  <body class="fo">
+    <!-- Sidenav -->
+    <nav
+      id="sidenav-3"
+      class="left-0 relative top-0 z-[1035] h-screen w-60 -translate-x-full overflow-hidden bg-zinc-800 shadow-[0_4px_12px_0_rgba(0,0,0,0.07),_0_2px_4px_rgba(0,0,0,0.05)] data-[te-sidenav-hidden='false']:translate-x-0"
+      data-te-sidenav-init
+      data-te-sidenav-hidden="false"
+      data-te-sidenav-color="white"
+    >
+      <ul
+        class="relative m-0 list-none px-[0.2rem] font-semibold"
+        data-te-sidenav-menu-ref
+      >
+        <li class="relative p-6 pr-16 pl-5">
+          <img src="logo.png" />
+        </li>
+        <li class="relative">
+          <a
+            class="flex h-12 cursor-pointer items-center truncate rounded-[5px] px-6 py-4 text-[0.875rem] text-gray-300 outline-none transition duration-300 ease-linear hover:bg-white/10 hover:outline-none focus:bg-white/10 focus:outline-none active:bg-white/10 active:outline-none data-[te-sidenav-state-focus]:outline-none motion-reduce:transition-none"
```

Comment:
@Bishakha1589 This div contains a lot of unnecessary classes, please identify them and remove them. Keep your code as clean as possible. Please check all other divs for classes too that don't do anything in the ui